### ISSUE TITLE
Slow startup of myMPD on large collections

### ISSUE THREAD

**Issue by skidoo23** [jcorporation/myMPD#913]
myMPD version: [e.g. 8.1.0]
myMPD 10.1.4 git devel and MPD 0.24 from git master.

**Describe the bug**
Startup of myMPD takes ages and results in high cpu load on a single core. Lots of queries to MPD server.

**To Reproduce**
MPD up and running. Collection of more than 100k files.
`# systemctl restart mympd`

```
mpd.log
Dec 02 02:14 : client: [12] process command "search file "" window 72000:74000"
Dec 02 02:15 : client: [12] command returned 0
Dec 02 02:15 : client: [10] process command "status"
Dec 02 02:15 : client: [10] command returned 0
Dec 02 02:15 : client: [10] process command "idle"
Dec 02 02:15 : client: [10] command returned 1
Dec 02 02:15 : client: [10] process command "status"
Dec 02 02:15 : client: [10] command returned 0
Dec 02 02:15 : client: [10] process command "idle"
Dec 02 02:15 : client: [10] command returned 1
Dec 02 02:15 : client: [12] process command "search file "" window 74000:76000"
```
**Expected behavior**
Fast startup of myMPD in less than one second.

**Screenshots**
N/A

**Server plattform (please complete the following information):**
 - OS: Debian
 - OS version: 12
 - Plattform: x86_64
 - MPD version: 0.24

**Client plattform (please complete the following information):**
 - Device: Notebook 
 - OS: Windows 10 x86_64
 - Browser Firefox ESR (x86_64)
 - Browser version 102.5

**Debug logs (please attach if it can be useful)**
See above.
 
**Configuration (please attach if it can be useful)**
If required will supply later.

**Additional context**
N/A

**Comment by jcorporation** [comment]
This is a normal behavior. myMPD creates the album cache from the mpd database. This is a background task.

**Comment by skidoo23** [comment]
But why a full update it at every myMPD start?

**Comment by jcorporation** [comment]
myMPD does not save the album cache between restarts.

**Comment by skidoo23** [comment]
myMPD startup takes about 6 minutes on my Intel Atom D410 512K Cache, 1.66 GHz cpu. myMPD connects to MPD via socket. About 120k tracks - mostly mp3 and flac - on local storage.

**Comment by jcorporation** [comment]
Commit a0be5cdbd6e4377eed7043193d929c1798732213 increases the max responses from mpd to 4000. Can you test the devel branch?

**Comment by jcorporation** [comment]
That’s great, let me look if this could be improved further.

**Comment by jcorporation** [comment]
Are mpd stickers enabled in your setup?

**Comment by skidoo23** [comment]
sticker_file is commented out #in mpd.conf and sticker.sql file does not exist.

**Comment by skidoo23** [comment]
Switched to branch v10.2.0,
```
$ ./build.sh --release
# -/build.sh --install
# systemctl daemon-reload
# systemctl restart mympd
```

Now myMPD creates /var/cache/mympd/album_cache.json but it still blocks MPD for about three minutes with
```
...
Dec 07 02:27 : client: [9] process command "search file "" window 25000:30000"
Dec 07 02:27 : client: [9] command returned 0
Dec 07 02:27 : client: [9] process command "search file "" window 30000:35000"
Dec 07 02:27 : client: [9] command returned 0
Dec 07 02:27 : client: [9] process command "search file "" window 35000:40000"
Dec 07 02:27 : client: [9] command returned 0
Dec 07 02:27 : client: [9] process command "search file "" window 40000:45000"
Dec 07 02:27 : client: [9] command returned 0
Dec 07 02:27 : client: [9] process command "search file "" window 45000:50000"
Dec 07 02:27 : client: [9] command returned 0
...
```

**Comment by jcorporation** [comment]
Implementation is just not yet finished.

**Comment by skidoo23** [comment]
Please excuse my impatience ;)

**Comment by jcorporation** [comment]
> Please excuse my impatience ;)

Now there are all features implemented in the v10.2.0 branch. It is only roughly tested.
